src/main/java/com/internet/shop/controller/InjectDataController.java

```diff
@@ -27,11 +29,15 @@
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
-        User bob = new User("Bob", "bob123", "passBob");
-        User alice = new User("Alice", "alice123", "passAlice");
-        User userDave = new User("Dave", "dave3459", "passDave");
-        User userCharlie = new User("Charlie", "charlie777", "passCharlie");
+        User bob = new User("Bob", "bob123", "passBob", Set.of(Role.of("USER")));
+        User mainAdmin = new User("mainAdmin", "admin123", "admin123",
+                Set.of(Role.of("ADMIN"), Role.of("USER")));
```

Comment:
```suggestion
        User mainAdmin = new User("mainAdmin", "admin123", "admin123",
                Set.of(Role.of("ADMIN"))));
```

Reply:
I wanted to check situation when person has all accesses